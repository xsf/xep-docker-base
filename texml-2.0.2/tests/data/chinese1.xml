<TeXML>
	<cmd name="documentclass" nl2="1">
		<parm>article</parm>
	</cmd>
	<cmd name="usepackage" nl2="1">
		<opt>encapsulated</opt>
		<parm>CJK</parm>
	</cmd>
	<cmd name="usepackage" nl2="1">
    <parm>ucs</parm>
  </cmd>
	<cmd name="usepackage" nl2="1">
		<opt>utf8x</opt>
		<parm>inputenc</parm>
	</cmd>
	<env name="document">
		<env name="CJK">
			<parm>UTF8</parm>
			<parm>cyberbit</parm>
			&#x4E16;&#x754C;&#xFF0C;&#x4F60;&#x597D;&#xFF01;
		</env>
	</env>
</TeXML>
