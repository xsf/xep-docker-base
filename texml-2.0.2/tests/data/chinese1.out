\documentclass{article}
\usepackage[encapsulated]{CJK}
\usepackage{ucs}
\usepackage[utf8x]{inputenc}
\begin{document}
\begin{CJK}{UTF8}{cyberbit}
^^e4^^b8^^96^^e7^^95^^8c^^ef^^bc^^8c^^e4^^bd^^a0^^e5^^a5^^bd^^ef^^bc^^81
\end{CJK}
\end{document}
